week1_blog/index.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.0/css/all.min.css" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>인영이의 블로그</title>
+</head>
+<body>
+    <header>
+        <span class="icon">🍅</span>
+        <h2>인영이의 블로그</h2>
```

Comment:
```suggestion
        <h1>인영이의 블로그</h1>
```
페이지의 제목을 타나내는 부분이니 h1이 더 적합할듯하네요~
추가로 heading태그는 h1, h2, h3, ... 이런식으로 위계질서에 맞춰 선언해주셔야 접근성에 더 유리합니다 !